packages/block-library/src/social-link/edit.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { URLPopover } from '@wordpress/block-editor';
+import { useState } from '@wordpress/element';
+import {
+	Button,
+	IconButton,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getIconBySite } from './social-list';
+
+const SocialLinkEdit = ( { attributes, setAttributes, isSelected } ) => {
+	const { url, site } = attributes;
+	const [ showURLPopover, setPopover ] = useState( true );
```

Comment:
I found it unusual that this defaults to `true`. When tabbing into one of the child blocks the popover opens unexpectedly and tabbing is then constrained.

I understand that this is prompting the user to add a URL, but I don't think it's a good pattern from an accessibility point of view.

Reply:
Yep, I think this seems good. Thanks.